paper/paper.md

```diff
@@ -53,44 +53,56 @@ bibliography: paper.bib
+
+[comment]: <> (saving below sentences in case they are useful down the line)
+[comment]: <> (The entire framework flowchart is visualized in \autoref{fig:flowchart}. DeepRank2 software aims at unifying previously developed DL frameworks for data mining PPIs (DeepRank [@renaud_deeprank_2021], DeepRank-GNN [@reau_deeprank-gnn_2022]), and proteins' variants (DeepRank-Mut [@]). )
+[comment]: <> (Additionally, the software allows for much greater flexibility, allowing users to easily tailor the framework to specific patterns of interest and features, and select the pipeline's steps that best suits their requirements.)
+
 
 [comment]: <> (CHECK FOR AUTHORS: Do the authors clearly state what problems the software is designed to solve and who the target audience is?)
 [comment]: <> (CHECK FOR AUTHORS: Do the authors describe how this software compares to other commonly-used packages?)
 
 # State of the field
 
 [comment]: <> (Motivation for using 3D protein structures)
-Individual proteins' and protein complexes' 3D structures provide fundamental information to decipher biological processes at the molecular scale. Gaining knowledge on how those biomolecules interact in 3D space is key for understanding their functions and exploiting or engineering these molecules for a wide variety of purposes such as drug design [@GANE2000401], immunotherapy [@sadelain_basic_2013], or designing novel proteins [@nonnaturalppi]. For example, PPI data can be harnessed to address critical challenges in the computational prediction of peptides presented on the major histocompatibility complex (MHC) protein, which play a key role in T-cell immunity. Protein structures can also be exploited in molecular diagnostics for the identification of missense variants, that are pathogenic sequence alterations in patients with inherited diseases.
+The 3D structure of proteins and protein complexes provides fundamental information to understand biological processes at the molecular scale. Exploiting or engineering these molecules is key for many biomedical applications such as drug design [@GANE2000401], immunotherapy [@sadelain_basic_2013], or designing novel proteins [@nonnaturalppi]. For example, PPI data can be harnessed to address critical challenges in the computational prediction of peptides presented on the major histocompatibility complex (MHC) protein, which play a key role in T-cell immunity. Protein structures can also be exploited in molecular diagnostics for the identification of SRVs, that are pathogenic sequence alterations in patients with inherited diseases [$$$ref_needed].
+[comment]: <> ($$$ The second sentence repetitive with the previous sentence and the abstract. I actually still think it's a bit too repetitive like this.)
```

Comment:
Here it's not clear to which sentences you're referring to

Reply:
I meant from the original.
"Gaining knowledge on how those biomolecules interact in 3D space is key for understanding their functions and .." 
is pretty much the same as 
"Individual proteins' and protein complexes' 3D structures provide fundamental information to decipher ..."
which again is largely the same as 
"The 3D structure of proteins and protein complexes provides fundamental information to understand biological processes at the molecular scale" (from summary)